cmd/geth/main.go

```diff
@@ -169,6 +169,8 @@ var (
 		utils.L1DeploymentBlockFlag,
 		utils.CircuitCapacityCheckEnabledFlag,
 		utils.RollupVerifyEnabledFlag,
+		utils.ShadowforkFlag,
```

Comment:
It's probably different, but it might be worth to take a look at how Ethereum does shadowforks: https://github.com/ethpandaops/ethereum-package?tab=readme-ov-file#shadowforking

Reply:
Ethereum shadowforks are a CL thing AFAICS, so it is less relevant for us.